app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,19 @@
+class SessionsController < ApplicationController
+  def new; end
+
+  def create
+    user = User.find_by(email: params[:session][:email].downcase)
+    if user&.authenticate(params[:session][:password])
```

Comment:
giải thích cú pháp `&. ` nghe coi

Reply:
cái này lúc trước là use & use.authenticate.... Khi chạy rubocop thì nó bảo lỗi và gợi ý xài user&.authenticate,nên em làm theo để fix lỗi.

Reply:
rứa chú hơm tìm hiểu về cú pháp này ạ :p 
https://viblo.asia/p/the-safe-navigation-operator-in-ruby-ORNZq3LN50n

Reply:
Đây là tính năng từ bản Ruby 2.3 trở lên,được gọi là điều hướng an toàn.Khi muốn chương trình chạy an toàn và không bị lỗi ( xử lý với nil ),thì ta viết code khá dài dòng.Cú pháp &. giúp ta viết gọn lại.Em chỉ dừng lại ở mức độ hiểu code chỗ này thôi,còn nếu vận dụng để làm thì em chưa làm được.